src/visualizr/settings.py

```
@@ -1,147 +1,146 @@
+    step_t: int = 50
+    seed: int = 0
+    motion_dim: int = 20
+
+    image_path: FilePath = Field(default=None)
+    audio_path: FilePath = Field(default=None)
+
+    control_flag: bool = True
+    pose_driven_path: str = "not_supported_in_this_mode"
+    image_size: int = 256
+    device: Literal["cuda", "cpu"] = "cuda" if is_available() else "cpu"
+    motion_dim: int = 20
```

Ruff (0.12.2)

Class field `motion_dim` is defined multiple times

Remove duplicate field definition for `motion_dim`

(PIE794)